src/cert_context.rs

```diff
@@ -44,6 +43,32 @@ impl CertContext {
         }
     }
 
+    /// Decodes a PEM-formatted X509 certificate.
+    pub fn from_pem(pem: &str) -> io::Result<CertContext> {
+        unsafe {
+            assert!(pem.len() <= winapi::DWORD::max_value() as usize);
+
+            // the docs claim that you can pass null as the out pointer and
```

Comment:
I cannot reproduce that:
https://github.com/steffengy/schannel-rs/commit/12e010b0917df698fb3479c1bf1f5df26f5532dc
works fine locally.
Also AppVeyor tests passed:
https://ci.appveyor.com/project/steffengy/schannel-rs/build/1.0.104

Can you confirm that this commit doesn't work for you locally?